R/aaa-auto.R

```diff
@@ -852,7 +852,7 @@ distances_johnson_impl <- function(graph, from=V(graph), to=V(graph), weights) {
   res
 }
 
-distances_floyd_warshall_impl <- function(graph, from=V(graph), to=V(graph), weights=NULL, mode=c("out", "in", "all", "total"), method) {
+distances_floyd_warshall_impl <- function(graph, from=V(graph), to=V(graph), weights=NULL, mode=c("out", "in", "all", "total"), method=AUTOMATIC) {
```

Comment:
@Antonov548 This will need fixing (although currently it is unused). Translation needs to be set up for `igraph_floyd_warshall_algorithm_t`, or code generation needs to be disabled for this function (which is what I recommend now).

Reply:
I disabled codegen for this function, see last commit.